enterprise/kernel/src/main/java/org/neo4j/kernel/impl/store/format/highlimit/PropertyRecordFormat.java

```diff
@@ -34,7 +34,7 @@
  */
 class PropertyRecordFormat extends BaseOneByteHeaderRecordFormat<PropertyRecord>
 {
-    private static final int RECORD_SIZE = 48;
+    private static final int RECORD_SIZE = 64;
```

Comment:
This doesn't change the `PropertyType.getPayloadSizeLongs()` as far as I can see, so this is just wasting space.